### DISCUSSION TITLE
Can I see the items and tag them?

### DISCUSSION THREAD

**Discussion by KalmanG** [meilisearch/product#737] [Feedback & Feature Proposal]
I want to create an improved site search without editing a website...

I want to, for example,tag pages with what "type," of content it is. 

Is this sort of tagging possible to do with Meilisearch, or do I need to do the tagging where the original content sits?

**Comment by macraig** [comment]
Hi @KalmanG, can you expand on what you mean exactly by original content? You can definitely add a tag field to your Meilisearch documents.
